### PR TITLE
Prerender API Endpoints

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds content about API endpoints for Dynamic Prerendering. It will probably be unnecessary once the API References are public.

Reviewers - Please ignore the content in index.md, it is covered in another PR.

### CODE REVIEW

File: docs/_photoniq/prerendering/prerendering-tasks-api.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+- **Method:** POST
+- **Headers:**
+  - **`x-photoniq-customer-id`**: The unique identifier for the customer. (Required)
+  - **`x-photoniq-prerender-origin`**: The prerender origin. (Required)
+- **Request Body (JSON):**
+
+    ```json
+    jsonCopy code
+    {
```
Comment (ghost) [inline docs/_photoniq/prerendering/prerendering-tasks-api.md:65]: {
      "_key": "string",
      "origin": "string",
      "siteMaps": ["string"],
      "ttl": "string",
      "caching": boolean,
      "allowOriginHeaderValue": "string",
      "identifyNotFound":"string",
      "passThroughHeaders":Array,
      "addResponseHeaders":Object,
      "created": Date,
      "lastUpdate": Object
    }
    ```
    
    We have updated origin object @diana-macrometa

File: docs/_photoniq/prerendering/prerendering-tasks-api.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+**Endpoint:** `/api/prerender/v1/origin`
+
+- **Method:** PATCH
+- **Headers:**
+  - **`x-photoniq-customer-id`**: The unique identifier for the customer. (Required)
+  - **`x-photoniq-prerender-origin`**: The prerender origin. (Required)
+- **Request Body (JSON):**
+
+    ```json
+    jsonCopy code
```
Comment (ghost) [inline docs/_photoniq/prerendering/prerendering-tasks-api.md:95]: ⬆️ same as the above

**Review by ghost — CHANGES_REQUESTED** [review]
let's change the `origin` object and that is it

File: docs/_photoniq/prerendering/prerendering-tasks-api.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/_photoniq/prerendering/prerendering-tasks-api.md:65]: Updated

File: docs/_photoniq/prerendering/prerendering-tasks-api.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/_photoniq/prerendering/prerendering-tasks-api.md:95]: Updated

**Comment by diana-macrometa** [comment]
@tushar-macrometa, which endpoint did you want me to remove?

**Review by ghost — APPROVED** [review]
👍

**Comment by diana-macrometa** [comment]
@dlozina-macrometa and @tushar-macrometa - Which endpoint needed to be removed from the docs?

File: docs/_photoniq/prerendering/prerendering-tasks-api.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    ```
+
+- **Responses:**
+  - **`200`**: Cache purged successfully
+  - **`400`**: Invalid origin provided
+  - **`500`**: Internal server error
+
+## Purge CDN Cache
+
+**Endpoint:** `/api/prerender/v1/purge/cdn`
```
Comment (ghost) [inline docs/_photoniq/prerendering/prerendering-tasks-api.md]: @diana-macrometa, we need to remove this endpoint `/api/prerender/v1/purge/cdn`

**Review by ghost — DISMISSED** [review]
small change

File: docs/_photoniq/prerendering/prerendering-tasks-api.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/_photoniq/prerendering/prerendering-tasks-api.md]: Updated.

Until we get the API Reference into the documentation and public, we will need to update this topic with each new user-facing change.

**Review by elof — APPROVED** [review]
:shipit:

**Comment by tushar-macrometa** [comment]
> @tushar-macrometa, which endpoint did you want me to remove?

@diana-macrometa  - I wanted to remove Purge CDN cache endpoint but I can see that you already removed it.

**Review by tushar-macrometa — APPROVED** [review]

**Pull request merged**